app/views/layouts/pages.html.erb

```diff
@@ -32,5 +32,7 @@
     </main>
 
     <%= render 'shared/footer' %>
+
+    <%= javascript_include_tag 'pages', 'data-turbolinks-track' => true, 'data-turbolinks-eval' => false %>
```

Comment:
Since we're not using turbolinks for this page, we can actually just make the script

```erb
<%= javascript_include_tag 'pages' %>
```